services/generic/battery.md

```diff
@@ -1,44 +1,68 @@
 
-Type | Interface          | Value type | Properties | Description
------|--------------------|------------|----------- |------------------
-in   | cmd.lvl.get_report | null       |            | Get battery level over level report.
-out  | evt.lvl.report     | int        | state      |
--|||
-out  | evt.alarm.report   | str_map    |            | val = {"event": "low_battery", "status": "activ"}
--|||
-in   | cmd.health.get_report| null     |            | Request battery health report
-out  | evt.health.report  | int        |            | Battery health in %
-in   | cmd.sensor.get_report | string  |            | Request battery temperature report in celsius.
-out  | evt.sensor.report  | float      | unit       | Battery temperature in celsius.
--|||
-in   | cmd.battery.get_report | null |              | Get full battery report
-out  | evt.battery.report | object |                | Battery full report
+| Type | Interface          | Value type | Storage | Description                                                                                                                                          |
+|------|--------------------|------------|---------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| in   | cmd.lvl.get_report | null       |         | Requests the battery level report.                                                                                                                   |
+| out  | evt.lvl.report     | int        |         | Reports the battery level as percentage.                                                                                                             |
+| -    |                    |            |         |                                                                                                                                                      |
+| out  | evt.alarm.report   | str_map    | `event` | Reports [`event`](#definitions) status. See [`event_report`](/services/generic/alarm.md#definitions) definition from alarm service for more details. |
 
+## Service properties
 
-Battery report object example :
+| Name         | Type      | Example                                        | Description                                                                            |
+|--------------|-----------|------------------------------------------------|----------------------------------------------------------------------------------------|
+| `sup_events` | str_array | ["low_battery", "replace_soon", "replace_now"] | List of supported `event` values. See the list of [well-defined events](#definitions). |
 
-```javascript
+## Definitions
 
-{
-   "lvl": 90 ,
-   "health" : 70,
-   "state" : "charging",
-   "temp_sensor" : 40,
-}
+* `event` represents a battery notification and is one of: `low_battery`, `replace_soon`, `replace_now`, `charging`, `charged`, `charge_soon`, `charge_now`.
```

Comment:
'low_battery' and 'replace_soon' are duplications... can we remove them if not a legacy issue ?

Reply:
`replace_soon` is native Z-Wave notification, while `low_battery` is custom one. We need both even though they might be almost the same.